solutions/recursion/recursive-sequence.js

```diff
@@ -0,0 +1,38 @@
+let fs = require("fs");
+let data = fs.readFileSync(0, 'utf-8');
+let idx = 0;
+data = data.split('\n');
+
+function readLine() {
+	idx++;
+	return data[idx - 1];
+}
+
+// -------- Do NOT edit anything above this line ----------
+// Use readLine() for taking input, it will read one line of from the input  and is stored in string format
+
+function f(operands, num, n){
```

Comment:
@prateek-mehra 
1. We don't need the parameters `operands` and `num`. Just `n` is sufficient.
2. `currentTerm` will be a product of `n` integers, starting from (n*(n-1)/2)
3. This implementation can be rewritten as `currentTerm + f(n-1)`.
4. Also, please format the code properly. There are extra empty lines, extra indentation etc. If you are using VS Code, you can use the built-in formatter. Right click -> Format Document